backend/api/main.py

```diff
@@ -12,7 +12,7 @@
     FullSchoolDistrict,
     BaseMoodleSettings,
     FullMoodleSettings,
-    MoodelUser
+    MoodelUser,
```

Comment:
Update `MoodelUser` to `MoodleUser` throughout.

Reply:
I vaguely recall Moodle having some unique spelling for certain words - is this one of those cases where we are mirroring their spelling or was it a typo?